gapic/samplegen/samplegen.py

```diff
@@ -664,22 +662,22 @@ def _validate_loop(self, loop):
 def generate_sample(sample,
                     id_is_unique: bool,
                     env: jinja2.environment.Environment,
-                    api_schema: api.API) -> Tuple[str, str]:
+                    api_schema: api.API) -> str:
     """Generate a standalone, runnable sample.
 
     Rendering and writing the rendered output is left for the caller.
 
     Args:
         sample (Any): A definition for a single sample generated from parsed yaml.
-        id_is_unique (bool): Indicates whether sample["id"] globally unique.
+        id_is_unique (bool): Indicates whether the sample's id is unique.
+                             If it is not, sample["id"] is modified
+                             as an output parameter.
```

Comment:
It's trying to indicate that `generate_sample` modifies its inputs; since the sample is passed by reference, and `generate_sample` modifies `sample["id"]`. Calling something an output parameter, or in this case an in/out parameter, is a C/C++ convention.

Reply:
Obviated after subsequent changes to ID disambiguation.